string wheel
float32 PWM
float32 targetVel
float32 encoderVel
